### PR TITLE
Update Wanna Fly to v1.0.3

### PR DESCRIPTION
**Pull request by Pe241**
### Changed

- Added more one portal that makes it easier to go up

- Added portal protect for players

- Fixed tip message

### CODE REVIEW

File: Wanna Fly/map.xml [hunk 1]
```diff
@@ -140,12 +144,26 @@
...
+      <cuboid min="55,11,10" max="59,14,14"/>
+      <cuboid min="23,11,42" max="27,14,46"/>
+      <cuboid min="23,24,10" max="27,26,14"/>
+    </union>
     <apply block-break="never" region="everywhere"/>
+    <apply kit="tp" region="portals"/>
   </regions>
   <broadcasts>
-    <tip after="1s">Hey guys! You can fly like a creative mode in this map! Try it!</tip>
+    <tip after="0s">Hey guys! You can fly like in a creative mode! Try it!</tip>
```
Comment (KingOfSquares) [inline Wanna Fly/map.xml:166]:
```suggestion
    <tip after="0s">This map has flying enabled, try it out!</tip>

```

`in a creative mode!` would be incorrect, posted a suggestion above.

Comment (Pe241) [reply Wanna Fly/map.xml:166]: Is also rule message incorrect?

**Comment by Pe241** [comment]
Fixed rule & tip messages.

**Comment by KingOfSquares** [comment]
Sorry didnt see your comment, rule message seems fine to me.

**Pull request merged**